core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -182,6 +189,7 @@ public enum SqlConformanceEnum implements SqlConformance {
     case HIVE:
     case BIG_QUERY:
     case SNOWFLAKE:
+    case SPARK:
```

Comment:
As we have changes for SortBy as well, please add test case for it as well.

Reply:
Added TCs for SortBy..